tests/system/action/motion_category/test_delete.py

```diff
@@ -0,0 +1,24 @@
+from openslides_backend.shared.exceptions import DatabaseException
+from tests.system.action.base import BaseActionTestCase
+from tests.util import get_fqid
+
+
+class MotionCategorySystemTest(BaseActionTestCase):
+    def test_delete_correct(self) -> None:
+        self.create_model(get_fqid("motion_category/111"), {"name": "name_srtgb123"})
+        response = self.client.post(
+            "/", json=[{"action": "motion_category.delete", "data": [{"id": 111}]}],
+        )
+        self.assertEqual(response.status_code, 200)
+        model = self.datastore.get(
+            get_fqid("motion_category/111"), get_deleted_models=2
+        )
+        assert model.get("meta_deleted")
+
+    def test_delete_wrong_id(self) -> None:
+        self.create_model(get_fqid("motion_category/112"), {"name": "name_srtgb123"})
+        with self.assertRaises(DatabaseException):
+            self.client.post(
+                "/", json=[{"action": "motion_category.delete", "data": [{"id": 111}]}],
+            )
+        self.assert_model_exists(get_fqid("motion_category/112"))
```

Comment:
Please add a test with a motion which has a category assigned, which is the deleted, to make sure the relation is properly updated

Reply:
Done.